src/components/Axis/Ticks.tsx

```diff
@@ -10,12 +10,14 @@ interface CoordinatesXY {
 
 export interface TickProps {
   style: CSSProperties;
-  children: ReactNode;
+  children?: ReactNode;
 
   line: CoordinatesXY;
   text: Omit<CoordinatesXY, 'x2' | 'y2'>;
+  secondary?: boolean;
 
   strokeColor?: string;
+  strokeHight?: number;
```

Comment:
Small spelling mistake. Should be strokeHeight. I'm using a plugin in vscode : 'Code Spell Checker' to highlight those kind of errors.